spot_price_monitor/spot_price_monitor.py

```diff
@@ -129,13 +142,17 @@ def get_args():
 
 def update_spot_price_metrics(metric, prices):
     ''' Updates prometheus gauge based on input list of prices'''
+
     for price in prices:
         metric.labels(
             instance_type=price['InstanceType'],
             availability_zone=price['AvailabilityZone']
             ).set(price['SpotPrice'])
 
+
 def update_ondemand_price_metrics(metric, prices, types, zones):
+    ''' Updates ondemand prometheus gauge based on input list of prices'''
```

Comment:
Nitpick; would probably phrase this as "Updates a Prometheus gauge with on demand prices" or something. What do you think?